README.md

```diff
@@ -0,0 +1,48 @@
+## Getting Started
+
+Refer to the [Yesod Quickstart guide](https://www.yesodweb.com/page/quickstart) for install instructions.
+
+## Database Setup
+
+After installing Postgres, run:
+
+```
+createuser PROJECTNAME_LOWER --password PROJECTNAME --superuser
+createdb PROJECTNAME_LOWER
+createdb PROJECTNAME_LOWER_test
+```
+
+## Development
+
+Start a development server with:
+
+```
+stack exec -- yesod devel
+```
+
+As your code changes, your site will be automatically be recompiled and redeployed to localhost.
+
+## Tests
+
+```
+stack test --flag PROJECTNAME_LOWER:library-only --flag PROJECTNAME_LOWER:dev
+```
+
+(Because `yesod devel` passes the `library-only` and `dev` flags, matching those flags means you don't need to recompile between tests and development, and it disables optimization to speed up your tests).
```

Comment:
@snoyberg Is there a way Stack can default `stack test` to use these flags, so they don't need to be specified on the command line?

Reply:
Unfortunately not. A somewhat funny approach we could consider is adding a `Makefile` with targets like `devel` and `test`. I've seen a number of people doing this recently.

Reply:
Hm ok, something to consider then